nci/workspace_cleaner.rb

```diff
@@ -82,9 +82,21 @@ def cleanup?(workspace)
         puts '  ws-cleanup => delete'
         return true
       end
+      # delete management jobs when they become old
```

Comment:
There is no constraint on mgmt anywhere?

Reply:
improved the docs..